jest.config.js

```diff
@@ -19,12 +19,14 @@
  */
 
 module.exports = {
-  collectCoverageFrom: ['src/**/*.js'],
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  collectCoverageFrom: ['src/**/*.{js,ts}'],
```

Comment:
added support for ts, but I think this won't work with the js tests that exist. so need to cleanup and make sure we are left with no js tests 😅